nginx/nginx.conf

```diff
@@ -0,0 +1,49 @@
+daemon off;
+worker_processes 1;
+error_log stderr warn;
+pid /tmp/nginx.pid;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    sendfile on;
+
+    upstream fastapi {
+        server localhost:8080;
+    }
+
+    map $http_authorization $bearer_token {
+        "~^Bearer\s+(.+)$" $1;
+        default "";
+    }
+
+    proxy_cache_path /var/cache/nginx levels=1:2 keys_zone=fastapi_cache:10m max_size=1g inactive=60m;
+
+    server {
+        listen 80;
+        server_name localhost;
+
+        location / {
+            proxy_pass http://fastapi;
+            proxy_set_header Host $host;
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header X-Forwarded-Proto $scheme;
+
+            proxy_cache fastapi_cache;
+
+            proxy_cache_key "$scheme$proxy_host$request_uri$bearer_token";
+
```

Comment:
I'll be fixing that on the next patch; it will follow what is done on the `sonata-cell-service`.
That service checks whether the user has permissions to the particular project before allowing access to the cache.

The one issue I'm looking at right is how to "easily" determine what project is being looked at: pulling it from the content URL is a bit frustrating.
On the `sonata-cell-service` side, it's passed in as a param, but perhaps that can't be done here.

Reply:
@kplatis here we are not authenticating the user, so "permissions" are not checked actually it's workaround that can be bypassed easily by generating jwt without the right encoding.